rcsbsearchapi/search.py

```diff
@@ -1268,13 +1447,20 @@ def _extract_identifiers(query_json: Optional[Dict]) -> List[str]:
 
     def _make_params(self, start=0):
         "Generate GET parameters as a dict"
-        return dict(
+        query_dict = dict(
             query=self.query.to_dict(),
             return_type=self.return_type,
             request_info=dict(query_id=self.query_id, src="ui"),  # "TODO" src deprecated?
             # v1 -> v2: pager parameter is renamed to paginate and results_content_type parameter added (which has a list as its value)
+            # request_options=dict(paginate=dict(start=start, rows=self.rows), results_content_type=self.return_content_type, results_verbosity=self.results_verbosity),
```

Comment:
Some more cleanup? (Is results_verbosity being added?)

Reply:
Was the issue with the scores for results verbosity fixed? I can't remember.

I'll delete these lines for now.